Stam_Exercise/stam_ex.py

```diff
@@ -0,0 +1,29 @@
+import sys
+
+FILE1 = 1
+FILE2 = 2
+
+
+def main():
+    try:
+        with open(sys.argv[FILE1], 'r') as f1:
+            with open(sys.argv[FILE2], 'w') as f2:
+                f_content = f1.read().split('\n')
+                print(f_content)
+
+                new_content = (list(map(lambda x: 'liel' + ' ' + x + ' ' + 'yaakobov', f_content)))
+                print(new_content)
+
+                for i in new_content:
+                    # print(i)
+                    f2.writelines("{}\n".format(i))
+
+        with open(sys.argv[FILE2], 'r') as f:
+            print(f.read())
+
+    except Exception as e:
+        print(e)
```

Comment:
If you catch an exception and don't do any thing with it (just print it as it is), then you don't need the try-except.
Either you want to catch the exception and do something with it or you don't want to do anything and just let the exception raise